CHANGELOG.md

```diff
@@ -4,16 +4,48 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## 1.5.0 (unreleased)
+## 1.5.0 (2019-10-31)
+
+This release adds major new features and bugfixes since the 1.4.2 release.
+We deem it moderate priority for upgrading.
+
+This release adds compression of hypertable chunks as a major new feature.
+
+This release also adds support for parallel query coordination to the
+ChunkAppend node. Previously ChunkAppend would rely on parallel coordination
+in the underlying scans for parallel plans.
 
 **For this release only**, you will need to restart the database before running
 `ALTER EXTENSION`
 
-**Features**
+**Major Features**
 * #1433 Make ChunkAppend parallel aware
+* #1434 Compression
```

Comment:
"#1434 Introducing native compression, multiple compression algorithms, and hybrid row/columnar projections"

(or something like that)

And major feature:

#1393 Moving chunks between different tablespaces